qdrant-landing/content/articles/discovery-search.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Discovery needs context" #required
+short_description: Discover points by constraining the space
+description: Qdrant released a new functionality that lets you constrain the space in which a search is performed, relying only pure vectors #required
+social_preview_image: /articles_data/discovery-search/social_preview.jpg # This image will be used in social media previews, should be 1200x630px. Required.
+small_preview_image: /articles_data/discovery-search/icon.svg # This image will be used in the list of articles at the footer, should be 40x40px
+preview_dir: /articles_data/discovery-search/preview # This directory contains images that will be used in the article preview. They can be generated from one image. Read more below. Required.
+weight: -100 # This is the order of the article in the list of articles at the footer. The lower the number, the higher the article will be in the list.
+author: Luis Cossío # Author of the article. Required.
+author_link: https://coszio.github.io # Link to the author's page. Required.
+date: 2023-12-11T13:00:00+03:00 # Date of the article. Required.
+draft: false # If true, the article will not be published
+keywords: # Keywords for SEO
+  - why use a vector database
+  - specialty
+  - search
+  - discovery
+  - state of the art
+  - vector search
+---
+
+When Christopher Columbus and his crew sailed to cross the Atlantic Ocean, they were not looking for America. They were looking for a new route to India, and they were convinced that the Earth was round. They didn't know anything about America, but since they were going west, they stumbled upon it.
+
+They couldn't reach their target, because the context of the geography didn't let them, but once they realized it wasn't India, they claimed it a new "discovery" for their crown. Let's keep this concepts of target and context in mind, as we explore the new functionality of Qdrant: __Discovery search__.
```

Comment:
> They couldn’t reach their target, because the context of the geography didn’t let them.

I am not sure this analogy helps me understand the concept of `context` here.
What is the context of geography exactly?

Reply:
The idea is to give a notion as to why sometimes it is important for a search to be tied to a context. I see now that saying "context of geography" is ambiguous. Let me rephrase.

Reply:
Upgraded it, @agourlay, is it clearer now?